pkg/dump/dump.go

```diff
@@ -907,6 +921,34 @@ func GetAllRBACREndpointPermissions(ctx context.Context,
 	return eps, nil
 }
 
+// GetAllLicenses queries Kong for all the Licenses using client.
+func GetAllLicenses(
+	ctx context.Context, client *kong.Client, tags []string,
+) ([]*kong.License, error) {
+	var licenses []*kong.License
+	opt := newOpt(tags)
+
+	for {
+		s, nextopt, err := client.Licenses.List(ctx, opt)
+		if kong.IsNotFoundErr(err) || kong.IsForbiddenErr(err) {
```

Comment:
We don't return an error when we get a forbidden status?

Reply:
After tests, Kong does not return 403 when accessing `GET /license` endpoint. So Forbidden should be an error. Updated.